src/AlgebraicGeometry/Schemes/ProjectiveSchemes/Objects/Constructors.jl

```diff
@@ -89,7 +89,7 @@ Multivariate polynomial ring in 3 variables over QQ graded by
 """
 function projective_space(A::Ring, var_symb::Vector{<:VarName})
   n = length(var_symb)
-  S, _ = graded_polynomial_ring(A, Symbol.(var_symb))
+  S, _ = graded_polynomial_ring(A, Symbol.(var_symb); cached=false)
```

Comment:
```suggestion
  S, _ = graded_polynomial_ring(A, Symbol.(var_symb))
```
Same for the constructors for `projective_space`